examples/example-emailtask/nodes/email.json

```
@@ -0,0 +1,34 @@
+{
+    "id": "48286e12-ba6c-4627-97fd-2296a20c9db1",
+    "name": "Send Email",
+    "description": "Send e-mail.",
+    "deserializeAs": "EmailTask",
+    "inputVariables": [
+      {
+        "key": "exampleEmailTo",
+        "type": "PROCESS"
+      },
+      {
+        "key": "email",
+        "type": "LOCAL",
+        "spin": true
+      },
+      {
+        "key": "timestamp",
+        "type": "PROCESS"
+      },
+      {
+        "key": "tenantId",
+        "type": "PROCESS"
+      }
+    ],
+    "outputVariable": {},
+    "includeAttachment": false,
+    "mailTo": "{{exampleEmailTo}}",
```

```suggestion
    "mailTo": "${exampleEmailTo}",
```
If we go with a process variable, then this change is needed.

---

done